tests/acceptance/home/with-repositories-test.js

```diff
@@ -56,9 +56,11 @@ moduleForAcceptance('Acceptance | home/with repositories', {
   }
 });
 
-test('the home page shows the repositories', (assert) => {
+test('the home page shows the repositories', function (assert) {
   sidebarPage.visit();
 
+  // test fails without this just by adding the feature flag :tableflip:
+  andThen(() => {});
```

Comment:
I wonder why this is happening to us 😞  maybe it’ll magically go away with the new test setup hehe